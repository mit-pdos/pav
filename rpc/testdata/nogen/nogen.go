package rpc

//rpc: no encode needed.
//rpc: no decode needed.
type arg struct {
	x uint64
}
