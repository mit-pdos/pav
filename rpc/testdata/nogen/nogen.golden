// Auto-generated from spec "github.com/mit-pdos/pav/rpc/testdata/nogen/nogen.go"
// using compiler "github.com/mit-pdos/pav/rpc".
package rpc

import (
	"github.com/mit-pdos/pav/marshalutil"
	"github.com/tchajed/marshal"
)
